app_dart/lib/src/request_handlers/github_webhook.dart

```diff
@@ -86,6 +98,30 @@ class GithubWebhook extends RequestHandler<Body> {
     }
   }
 
+  Future<void> _handleCreateRequest(String rawRequest, DatastoreService datastore) async {
```

Comment:
Avoid adding code to the `RequestHandler` classes as they're harder to test.

This should be moved into a `BranchService` class

Reply:
Sure, moved